### ISSUE TITLE
click in a link

### ISSUE THREAD

**Issue by darlanmendonca** [DevExpress/testcafe#1897]
### Are you requesting a feature or reporting a bug?
probably a bug, when I click in a link, nothing occur, and testcafe wait for a long time, until stop the process.

### What is the current behavior?
I'm trying to write a spec to click in a link `<a href="">`

### What is the expected behavior?
The browser need to goes to the url specified on link

#### Provide the test code and the tested page URL (if applicable)
```js
test('click in button create, redirect to /precificacoes/criar', async (page) => {
    await pricingSettings.clickInCreateButton()

    const location = await page.eval(() => window.location)

    await expect(location.pathname).to.equal('/precificacoes/criar')
})
```

where `clickInCreateButton` is defined in other file, using page object

```js
class PricingSettingsPageObject {
  this.createButton = selector('a.mn-button[ui-sref="app.pricingSettings.create"]')
  
  async clickInCreateButton() {
    await this.page.click(this.createButton)
  }
}
```

### Specify your

* operating system: mac OS
* testcafe version: 18.x
* node.js version: 8.x

**Comment by helen-dikareva** [comment]
I suppose the issue is specific to your page. Could you please provide us with an url of your page? If it is not possible, create a simple example that reproduces the problem.

**Comment by rumpl** [comment]
Hello,

Here is a test case that reproduces the problem:

```typescirpt
import { Selector } from 'testcafe';
import { AngularJSSelector } from 'testcafe-angular-selectors';

fixture`Getting Started`

fixture`Getting Started`
  .page`https://angularjs-awzrxp.stackblitz.io/`;

test('My first test', async t => {
  await t
    .wait(7000)
    .click('#link')
    .wait(4000)
    .expect(Selector('h1').innerText).contains('Second');
});
```

The code of the application is here: https://stackblitz.com/edit/angularjs-awzrxp

When a `<button>` contains a `ui-sref` nothing happens on click. However if it's an `<a>` the test passes.

**Comment by AlexanderMoskovkin** [comment]
@miherlosev @LavrovArtem I've reproduced this in the `testcafe-hammerhead playground`. Please take a look at this. Just open the https://stackblitz.com/edit/angularjs-awzrxp page under the playground and try to click the button. Nothing happens.

**Comment by miherlosev** [comment]
The fix is already in [PR](https://github.com/DevExpress/testcafe-hammerhead/pull/1439).

**Comment by LavrovArtem** [comment]
The fix is published in `testcafe@0.18.6-dev20180117`
